ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/request/heal/StorageFlatDatabaseHealingRangeRequest.java

```diff
@@ -56,7 +57,7 @@ public class StorageFlatDatabaseHealingRangeRequest extends SnapDataRequest {
   private final Bytes32 storageRoot;
   private final Bytes32 startKeyHash;
   private final Bytes32 endKeyHash;
-  private TreeMap<Bytes32, Bytes> slots;
+  private NavigableMap<Bytes32, Bytes> slots;
```

Comment:
did error prone complain about the TreeMaps?

Reply:
NavigableMap is the superclass of both TreeMap and the streaming class I used in the flat streaming impl (IIRC ConcurrentSkipListMap).  I have been slowly changing TreeMap references to NavigableMap where necessary.  The current diff for this PR doesn't show this as a diff but I wanted to address your q 👍